gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineExactDependencies.java

```diff
@@ -176,6 +157,9 @@ private static void configureSourceSet(
 
             explicitCompile.get().extendsFrom(implCopy);
 
+            Optional<Configuration> maybeCompile =
```

Comment:
I think as a follow up potentially, once we're out of the woods, we can delete this, I don't think it will ever run because that configuration doesn't exist anymore